src/main/java/com/google/devtools/build/lib/bazel/rules/java/jdk.WORKSPACE

```diff
@@ -117,6 +117,18 @@ maybe(
     ],
 )
 
+# This must be kept in sync with the top-level WORKSPACE file.
+maybe(
+    http_archive,
+    name = "remotejdk11_linux_ppc64le",
+    build_file = "@local_jdk//:BUILD.bazel",
+    strip_prefix = "jdk-11.0.7+10",
+    urls = [
+        "https://mirror.bazel.com/openjdk/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.2%2B9/OpenJDK11U-jdk_ppc64le_linux_hotspot_11.0.7_10.tar.gz",
+        "https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.2%2B9/OpenJDK11U-jdk_ppc64le_linux_hotspot_11.0.7_10.tar.gz",
```

Comment:
This URL doesn't seem to be valid and gives an HTTP 404. Can you replace it with the one above?

The difference is one character: This one `jdk-11.0.2%2B9` vs. the above `jdk-11.0.2%2B10`.